docs/helpers/env_vars.yaml

```
@@ -4357,6 +4510,26 @@ GATEWAY_STORAGE_REGISTRY_RULES:
   deprecationVersion: ""
   removalVersion: ""
   deprecationInfo: ""
+GATEWAY_STORAGE_SHARES_ENDPOINT:
+  name: GATEWAY_STORAGE_SHARES_ENDPOINT
+  defaultValue: com.owncloud.api.storage-shares
+  type: string
+  description: The endpoint of the storag-shares service. Can take a service name
```

```suggestion
  description: The endpoint of the storage-shares service. Can take a service name
```

Pretty sure that an "e" is missing. But that would need to be corrected in the original source.

---

What a catch... 👍 
I have fixed the string in the source and re-ran the make command.
Committed and pushed, Means this suggestion is now obsolete.